com.unity.multiplayer.mlapi/Runtime/Profiling/ProfilerConstants.cs

```diff
@@ -0,0 +1,21 @@
+namespace MLAPI.Profiling
+{
+    public enum ProfilerConstants
```

Comment:
I'm still not convinced about using an enum here instead of string constants - why bother using a Dictionary, if we're restricting the set of values anyways? I think we should go one way or the other, either make it an explicit data set, or make it extensible. Right now we're taking all the cost of a Dictionary (loose abstraction, heap alloc) and none of the benefits (extensibility)

Reply:
I do agree with what Beck says here. However, this can offer a clear distinction between standard and custom data to profile. Not sure if we're aiming for such support right now.